extensions/gdax/products.json

```diff
@@ -1,65 +1,74 @@
 [
   {
-    "id": "BTC-GBP",
```

Comment:
removing, as the update script does not include them, and I couldn't find any references to this id being in use anywhere.